dts/riscv/it8xxx2.dtsi

```diff
@@ -1396,7 +1402,8 @@
 			label = "pwm_6";
 			status = "disabled";
 			pwmctrl = <&prs>;
-			pinctrl-0 = <&pinctrl_pwm6>; /* GPA6 */
+			pinctrl-0 = <&pwm6_default>; /* GPA6 */
```

Comment:
Question: why is this configured at SoC files? Does that mean the SoC has a single pin configuration selection? Boards need to make the pinctrl selection, there should be no default in SoC files unless there’s a single routing possible.

Reply:
If the driver is enabled, most of them are configured with this selection. If we need to configure other selection, we will write in the board dts to override the SoC files(it8xxx2.dtsi) selection.
It is considered that the definition of each board is too repetitive, so it is defined in the SoC files.

Reply:
see my previous comment